server/core/config/raw/policies.go

```diff
@@ -27,10 +28,20 @@ func (p PolicySets) ToValid() valid.PolicySets {
 		policySets.Version, _ = version.NewVersion(*p.Version)
 	}
 
+	// Default number of required reviews for all policy sets should be 1.
+	policySets.ApproveCount = p.ApproveCount
+	if policySets.ApproveCount == 0 {
```

Comment:
What if it was less than 0? Should we check for that?

```suggestion
	if policySets.ApproveCount <= 0 {
```

Reply:
Good call. I've updated code to automatically correct for negative input. Though this will not throw an error and will silently succeed.